app/server/plugins/content.ts

```diff
@@ -1,13 +1,13 @@
 export default defineNitroPlugin((nitroApp) => {
-  nitroApp.hooks.hook('content:file:afterParse' as any, async (file) => {
+  nitroApp.hooks.hook('content:file:afterParse' as any, (file) => {
     // Filter out non-markdown files
     if (!file._id.endsWith('.md')) {
       return
     }
 
     // Set the icon for the file if it is not already set
     if (!file.icon) {
-      file.icon = await resolveIcon(file._path)
+      file.icon = resolveIcon(file._path)
```

Comment:
I intentionally kept it async to prepare for your external resolver 😅

Reply:
I will add it back! 😅